### ISSUE TITLE
Does not load websites

### ISSUE THREAD

**Issue by yoctocell** [luakit/luakit#850]
## Current Behavior

I just installed Luakit and when I start it I just get a blank screen and the statusbar says `https://luakit.github.io` on the left side and `(notrust) [1/1]` on the right side. It doesn't load any website, there is no CPU usage and it uses around 100 MB of memory.

This is the output I get in my terminal
```
$ luakit
[    0.035551] I [core/luah]: Loading rc: /etc/xdg/luakit/rc.lua
[    0.092806] I [lua/adblock]: found 0 filter lists
[    0.109720] I [lua/styles]: found 0 user stylesheets
[    0.111742] I [core/web_context]: Web process count: 4

** (luakit:6316): WARNING **: 11:22:44.214: webkit_web_context_set_web_process_count_limit is deprecated and does nothing. Limiting the number of web processes is no longer possible for security reasons
```

## Expected Behavior

The website should be loaded and there should be spike in CPU usage.

## Reproduction Instructions

Just install luakit and run it from your terminal.

## Environment

Linux Distribution & Version: Void Linux with glibc
Output of `luakit --version`:
```
$ luakit --version
luakit fc07ed5a
  built with webkit 2.22.6 (installed version: 2.28.2)
```

**Comment by neoromantique** [comment]
Ubuntu 20.04, also affected by this. 
```
~> luakit --version
luakit 2.0.0
  built with webkit 2.24.1 (installed version: 2.28.2)```

**Comment by craftyguy** [comment]
This is fixed on the master branch. IMHO there should be a release (#847...) tagged so the fix for this and many other issues (e.g. with newer webkitgtk..) is available for distros that package tagged releases.

**Comment by aidanholm** [comment]
The master branch is nearly three years old; do you mean the develop branch?

AFAIK develop is still busted; there's a WIP fix on the `wip-fix-804` branch but I haven't had much time to fix it.

**Comment by craftyguy** [comment]
Yes, sorry, I meant develop branch not master.

**Comment by enderger** [comment]
Having the same issue here on Gentoo testing as well.

**Comment by c0dev0id** [comment]
This issue is fixed with luakit 2.2. It's not yet a release but tagged already. Grab it from here: https://github.com/luakit/luakit/releases/tag/2.2

If your distribution package is older, please wait for an update. There's not much we can do about it here.
